src/mds/CDir.cc

```diff
@@ -2922,8 +2935,6 @@ void CDir::scrub_initialize()
     }
   }
   scrub_infop->directory_scrubbing = true;
-
-  assert(scrub_local()); // TODO: handle failure
```

Comment:
I still don't add this back in incremental patches. calling  scrub_local() twice for each dirfrag looks weird.